From 40ca409ca69d9f8b59ea623c43f936724212e8fc Mon Sep 17 00:00:00 2001
From: tnichols217 <62992267+tnichols217@users.noreply.github.com>
Date: Sat, 7 Sep 2024 19:29:19 -0400
Subject: [PATCH] add extra gradleflags flag

---
 nix/build-gradle-package.nix | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/nix/build-gradle-package.nix b/nix/build-gradle-package.nix
index b963c45..7953249 100644
--- a/nix/build-gradle-package.nix
+++ b/nix/build-gradle-package.nix
@@ -11,6 +11,7 @@
   lockFile ? null,
   # The Gradle package to use. Default is 'pkgs.gradle'.
   gradle ? null,
+  extraGradleFlags ? [],
   # Override the default JDK used to run Gradle itself.
   buildJdk ? null,
   # Override functions which fetch dependency artifacts.
@@ -102,8 +103,9 @@ let
       nativeBuildInputs = (args.nativeBuildInputs or [ ]) ++ [ finalAttrs.gradleSetupHook ];
 
       gradleFlags =
-        [ "--console=plain" ]
-        ++ lib.optional (finalAttrs.buildJdk != null) "-Dorg.gradle.java.home=${finalAttrs.buildJdk.home}";
+        # [ "--console=plain" ] ++
+        extraGradleFlags ++
+        lib.optional (finalAttrs.buildJdk != null) "-Dorg.gradle.java.home=${finalAttrs.buildJdk.home}";
 
       passthru =
         lib.optionalAttrs (offlineRepo != null) { inherit offlineRepo; } // (args.passthru or { });
-- 
2.45.2

